src/frontend/src/components/nav/MainMenu.tsx

```diff
@@ -63,7 +64,7 @@ export function MainMenu() {
         <Menu.Item
           icon={<IconLogout />}
           onClick={() => {
-            doClassicLogout();
+            doLogout(navigate);
```

Comment:
Breaking: Global Logout is not working anymore; if you log out and f5 refresh the page you are logged in again